lib/cli.js

```diff
@@ -32,7 +35,7 @@ if (process.argv.join('').indexOf('mocha') === -1) {
     if (!validateMessage(msg)) {
       var incorrectLogFile = (
         isFile
-        ? commitMsgFileOrText.replace('COMMIT_EDITMSG', 'logs/incorrect-commit-msgs')
+        ? commitMsgFileOrText.replace(filename, 'logs/incorrect-commit-msgs')
```

Comment:
@Garbee : `filename` will always be `GITGUI_EDITMSG`, but `GITGUI_EDITMSG` file will not be exist every time, when you are making commit form git gui this file is getting created and destroyed as well after the commit is success full.

This [line](https://github.com/kentcdodds/validate-commit-msg/pull/79/files#diff-f8849ae9f0614ba9464fdd14c2d98dc0R51) is checking if the file exists or not, while commit is made thru `cli`, that(`GITGUI_EDITMSG`) particular file does not exist and it will pick from the `COMMIT_EDITMSG` file. Hope it answers your question.

It is not breaking the current functionality, in our project user can commit from `cli` or `git gui` so it should work from tools. I have cross verified and create the PR